admin_guide/manage_users.adoc

```diff
@@ -0,0 +1,135 @@
+= Managing Users
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Adding Users
+
+Once new users log in to OpenShift, an account is created for that user and the
+cluster administrator can
+link:../admin_guide/manage_authorization_policy.html[manage the access level of
+each user].
+
+== Deleting Users
```

Comment:
It's nicer if we can avoid headings that just lead into a subheading. For example, if there's some intro or explanation we can add here before going into Improper Deletion.